src/prefect/server/services/task_run_recorder.py

```diff
@@ -176,8 +177,39 @@ async def record_task_run_event(event: ReceivedEvent, depth: int = 0):
     )
 
 
+async def record_lost_follower_task_run_events():
+    events = await causal_ordering().get_lost_followers()
+
+    for event in events:
+        await record_task_run_event(event)
+
+
+async def periodically_process_followers(periodic_granularity: timedelta):
+    """Periodically process followers that are waiting on a leader event that never arrived"""
+    logger.info(
+        "Starting periodically process followers task every %s seconds",
+        periodic_granularity.total_seconds(),
+    )
```

Comment:
Can this be a debug level log? My thinking is that this log will be useful to developers when debugging, but not useful to most users.
```suggestion
    logger.debug(
        "Starting periodically process followers task every %s seconds",
        periodic_granularity.total_seconds(),
    )
```

Reply:
yes definitely 👍 good call